packaging/installer/methods/alpine.md

```diff
@@ -21,16 +21,20 @@ cd netdata
 # build it, install it, start it
 ./netdata-installer.sh
 
-# make Netdata start at boot
-echo -e "#!/usr/bin/env bash\n/usr/sbin/netdata" >/etc/local.d/netdata.start
-chmod 755 /etc/local.d/netdata.start
+# make Netdata start at boot and stop at shutdown
+cat > /etc/init.d/netdata << EOF
+#!/sbin/openrc-run
 
-# make Netdata stop at shutdown
-echo -e "#!/usr/bin/env bash\nkillall netdata" >/etc/local.d/netdata.stop
-chmod 755 /etc/local.d/netdata.stop
+name="netdata"
+command="/usr/sbin/$SVCNAME"
 
-# enable the local service to start automatically
-rc-update add local
+depend() {
+        need net localmount
+        after firewall
+}
+EOF
 ```
 
+If you have installed Netdata in another directory, you have to change the content of the `command` variable in that script.
+
```

Comment:
Actually, none of this should be needed. We include an OpenRC init script as part of the regular install that _should_ be getting installed correctly on Alpine (I have not checked recently) as `/etc/init.d/netdata`.

Reply:
To clarify, I’m arguing that the whole section about manually setting up the startup/shutdown should be removed, not that we shouldn’t update what’s already there to use a proper init script instead of a local stub.

Reply:
There was no `/etc/init.d/netdata` file after following these instructions (installation done less than 1/2h before the PR), that's why I suggested the change...

Reply:
Ah, OK, that’s a bug then.

In that case, I’m all for adding this for now, but we should also open an issue about the fact that the OpenRC initscript is not being installed on Alpine.

Reply:
> In that case, I’m all for adding this for now, but we should also open an issue about the fact that the OpenRC initscript is not being installed on Alpine.

Kind reminder to do it if you think we need it!